src/push.rs

```diff
@@ -240,17 +240,25 @@ pub async fn build_profile(data: PushProfileData<'_>) -> Result<(), PushProfileE
     )
     .map_err(PushProfileError::ShowDerivationParse)?;
 
-    let &deriver = derivation_info
+    let deriver_key = derivation_info
         .keys()
         .next()
         .ok_or(PushProfileError::ShowDerivationEmpty)?;
 
-    let new_deriver = &if data.supports_flakes || data.deploy_data.merged_settings.remote_build.unwrap_or(false) {
+    // Nix 2.32+ returns relative paths (without /nix/store/ prefix) in show-derivation output
+    // Normalize to always use full store paths
+    let deriver = if deriver_key.starts_with("/nix/store/") {
+        deriver_key.to_string()
+    } else {
+        format!("/nix/store/{}", deriver_key)
```

Comment:
This doesn't allow locations for nix store other than /nix/store. One solution would be to look at $NIX_STORE_DIR env var for location of the store and prepend that, only defaulting to /nix/store. Another would be to utilize the output of `nix eval --expr builtins.storeDir`.